code/index.tsx

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const electron = require('electron');
+const app = electron.app;  // Module to control application life.
+const BrowserWindow = electron.BrowserWindow;  // Module to create native browser window.
+
+// Keep a global reference of the window object, if you don't, the window will
+// be closed automatically when the JavaScript object is garbage collected.
+var mainWindow = null;
+
+// Quit when all windows are closed.
+app.on('window-all-closed', function() {
```

Comment:
Writing the word function? ES5 called and they want their syntax back :P

`function() {}` => `() => {}`